main.go

```diff
@@ -63,13 +62,7 @@ func handleJobs(jobsC <-chan context.Job) {
 func executeTasks(tasks []*context.Task, watcher *context.Watcher) {
 	for _, task := range tasks {
 		command := task.Command
-		tokens := strings.Split(command, " ")
-		name := tokens[0]
-		var cmdArg []string
-		if len(tokens) > 1 {
-			cmdArg = tokens[1:]
-		}
-		cmd := exec.Command(name, cmdArg...)
+		cmd := exec.Command("/bin/bash", "-c", command)
```

Comment:
How about use of regexps? Not the final version of code, just the idea:
```go
var envVar = regexp.MustCompile(`\${([A-Za-z0-9._\-]+)}`)

func processCommand(cmd string) string {
	return envVar.ReplaceAllStringFunc(cmd, func(k string) string {
		return os.Getenv(envVar.ReplaceAllString(k, "$1"))
	})
}
```
And then update the existing code:
```diff
- 		command := task.Command
+ 		command := processCommand(task.Command)
		tokens := strings.Split(command, " ")
		name := tokens[0]
		var cmdArg []string
		if len(tokens) > 1 {
			cmdArg = tokens[1:]
		}
		cmd := exec.Command(name, cmdArg...)
```
After that all values in a `${SOME_NAME}` form will be replaced by the appropriate environment variables.